libs/ibl/CMakeLists.txt

```diff
@@ -35,21 +35,27 @@ include_directories(${PUBLIC_HDR_DIR})
 
 add_library(${TARGET} ${PUBLIC_HDRS} ${PRIVATE_HDRS} ${SRCS})
 target_include_directories(${TARGET} PUBLIC ${PUBLIC_HDR_DIR})
-
 target_link_libraries(${TARGET} math utils)
 
+add_library(${TARGET}_lite ${PUBLIC_HDRS} ${PRIVATE_HDRS} ${SRCS})
+target_compile_definitions(${TARGET}_lite PUBLIC -DFILAMENT_IBL_LITE=1)
+target_include_directories(${TARGET}_lite PUBLIC ${PUBLIC_HDR_DIR})
+target_link_libraries(${TARGET}_lite math utils)
+
 # ==================================================================================================
 # Compile options and optimizations
 # ==================================================================================================
 if (MSVC)
     target_compile_options(${TARGET} PRIVATE /fp:fast)
+    target_compile_options(${TARGET}_lite PRIVATE /fp:fast)
 else()
     target_compile_options(${TARGET} PRIVATE -ffast-math)
+    target_compile_options(${TARGET}_lite PRIVATE -ffast-math)
 endif()
 
 
 # ==================================================================================================
 # Installation
 # ==================================================================================================
-install(TARGETS ${TARGET} ARCHIVE DESTINATION lib/${DIST_DIR})
+install(TARGETS ${TARGET}_lite ARCHIVE DESTINATION lib/${DIST_DIR})
```

Comment:
Are we no longer distributing the full `ibl` library, only the `ibl_lite` one now? I think it's best to continuing distributing both for now, otherwise this could break clients (as it broke the iOS sample projects)

Reply:
Also we should call it ibl-lite not ibl_lite to be consistent with the other libraries. I noticed we have gltfio_core but all other libraries (.so, .jar, .aar, etc.) use dashes and not underscores.

Reply:
Added installation for both variants of the library, and renamed it to `ibl-lite`.  Note that `filamat_lite` uses an underscore.